src/ext-test/java/org/opentripplanner/ext/stopconsolidation/StopConsolidationParserTest.java

```diff
@@ -11,7 +11,7 @@ class StopConsolidationParserTest {
 
   @Test
   void parse() {
-    var file = ResourceLoader.of(this).file("consolidated-stops.csv");
+    var file = ResourceLoader.of(this).inputStream("consolidated-stops.csv");
```

Comment:
This variable is now a stream that needs to be closed.

Reply:
Adressed in f671f54e92444fa8533e5669f36ecd13ef00204e.